src/assembler.jl

```diff
@@ -35,10 +34,12 @@ Assembles the element matrix `Ke` into `a`.
 """
 function assemble!{T}(a::Assembler{T}, Ke::AbstractMatrix{T}, edof::AbstractVector{Int})
     n_dofs = length(edof)
```

Comment:
We could do something like:
`@assert checksquare(Ke) == length(edof)`
But if this doesn't hold, it will error in `sparse(I, J, V)` anyway. Thoughts?